.travis.yml

```diff
@@ -1,7 +1,8 @@
 language: node_js
 node_js:
-    - "6"
-    - "node"
+- 7 # Current stable
+- 6 # Active LTS until 2018-04-18
+- 4 # Active LTS until 2017-04-01
```

Comment:
I would argue that even though active LTS ends in April, support during maintenance is important (for a year, then). According to https://github.com/nodejs/LTS:

> Once a release moves into Maintenance mode, only ***critical*** bugs, ***critical*** security fixes, and documentation updates will be permitted.

So we may very well keep support for v4 an additional year, which is not really a big deal.